src/screens/SettingsScreen.tsx

```diff
@@ -56,47 +42,114 @@ export const SettingsScreen = () => {
             title={t('settings-profile-row-title', 'Profile')}
             action={() => navigate('Profile')}
           />
+          <Divider />
           <MainMenuItem
             title={account?.name || t('settings-account-selection', 'Accounts')}
             action={() => navigate('AccountSelection')}
           />
-          <View style={styles.subMenuContainer}>
-            <OAuthLogoutButton label={t('settings-logout', 'Logout')} />
-            <View style={styles.versionContainer}>
-              <Text testID={tID('version-text')}>
-                {t('settings-version', 'Version {{ version }}', {
-                  version: fullVersion,
-                })}
-              </Text>
-            </View>
-          </View>
+          <Divider />
         </ScrollView>
+        <View style={styles.subMenuContainer}>
+          <OAuthLogoutButton label={t('settings-logout', 'Logout')} />
+          <View style={styles.versionContainer}>
+            <Text testID={tID('version-text')}>
+              {t('settings-version', 'Version {{ version }}', {
+                version: fullVersion,
+              })}
+            </Text>
+          </View>
+        </View>
       </SafeAreaView>
     </View>
   );
 };
 
-const styles = StyleSheet.create({
-  mainMenuItem: {
+interface Props {
+  title: string;
+  action?: () => void;
+  badge?: React.ReactNode;
+}
+
+function MainMenuItem({ title, action, badge }: Props) {
+  const { styles } = useStyles('SettingsScreen', defaultStyles);
```

Comment:
Interesting- are there pros/cons to reusing a name & defaultStyles like this vs. us trying to be strict about each component getting its own name/defaultStyles?  I can't think of any major cons offhand looking at the `useStyles` impl, and dig the simplicity of this approach.

Reply:
Yes, I believe this is the correct approach.  

The purpose of our `useStyles` hook and the type-building we are doing is to create a single namespace of customizable style objects for every component that we export out of this library.  This also becomes part of the public API that can't be changed without major version bumps.  

As is, these sub-components can be refactored without breaking changes.  This also enables users to be able to customize a component by choosing from a set of style objects under a single namespace that is also the name of the component they are trying to style.

Reply:
Nice!  That's a huge pro I wasn't even thinking about - good deal.

Reply:
We want to make sure that we do this carefully though and may even want to consider a different approach. If we have a lot of these subcomponents that attach their style definition to a single namespace, it is likely there will be an overlap which can result in some less flexible styling abilities. For example, if `MainMenuItem` and `SettingsScreen` both defined a `container` style property, the consumer would only be able to provide one style override for this shared definition.

In your current use-case everything looks good though since the styles are only defined once in `defaultStyles` but it is definitely something to keep an eye on if we start splitting these shared definitions across multiple files.

Reply:
Additionally, we might consider allowing namespaces to use a dot notation to provide further granularity and grouping of component definitions, i.e. `SettingsScreen.MainMenuItem`

Reply:
Yea, for large components that are composed of many smaller components with their own files, I would think we would structure it in a way that made this very clear.  Likely using the filesystem to collect a component with it's subcomponents.

 But from the user's perspective, it is important that they don't have to dig around in the internals of the component to figure out how to style it.  We want to be able to produce a simple, straightforward API documentation for each component we export.